dcat/index.html

```diff
@@ -4108,9 +4108,9 @@ <h2>Security and Privacy</h2>
 
   <h2>Acknowledgments</h2>
   
-  <p>The editors gratefully acknowledge the contributions made to this document by <a href="https://www.w3.org/2000/09/dbwg/details?group=99375&public=1">all members of the working group</a>, especially Annette Greiner, Antoine Isaac, Dan Brickley, Ine de Visser, Jaroslav Pullmann, Linda van den Brink, Makx Dekkers, Nicholas Car, Rob Atkinson.</p>
+  <p>The editors gratefully acknowledge the contributions made to this document by <a href="https://www.w3.org/2000/09/dbwg/details?group=99375&public=1">all members of the working group</a>, especially Annette Greiner, Antoine Isaac, Dan Brickley, Ine de Visser, Jaroslav Pullmann, Lars G. Svensson, Linda van den Brink, Makx Dekkers, Nicholas Car, Rob Atkinson.</p>
```

Comment:
I do not remember which is the threshold for being considered in this list, nor if we have one. Anyway, 
Tom has [commented on some issues](https://github.com/w3c/dxwg/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Adcat+commenter%3Atombaker++ ), while Armin opened an [issue](https://github.com/w3c/dxwg/issues/438) and provided different [comments](https://github.com/w3c/dxwg/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Adcat+commenter%3Aarminhaller++).

```suggestion
  <p>The editors gratefully acknowledge the contributions made to this document by <a href="https://www.w3.org/2000/09/dbwg/details?group=99375&public=1">all members of the working group</a>, especially Annette Greiner, Antoine Isaac, Armin Haller, Dan Brickley, Ine de Visser, Jaroslav Pullmann, Lars G. Svensson, Linda van den Brink, Makx Dekkers, Nicholas Car, Rob Atkinson, Tom Baker.</p>
```

Reply:
Thanks, @riccardoAlbertoni . +1 from me.